app/helpers/sdxftp.py

```diff
@@ -42,12 +42,12 @@ def _connect(self):
     def deliver_binary(self, folder, filename, data):
         """Delivery binary delivers a single binary file to the given folder
         """
-        self.logger.info("Delivering binary file to FTP", host=self.host, folder=folder, filename=filename)
+        self.logger.info("Delivering binary file to FTP", host=self.host, folder=folder, filename=filename, tx_id=self.tx_id)
```

Comment:
Can we not bind the `tx_id` to the logger so we only need to specify it at the beginning?
```
self.logger.bind(tx_id=self.tx_id)
```

Reply:
I tried it and the number of log lines you get back when you search on the tx_id is more then you'd really want.  It's nicer when you see 'Delivering/Delivered file' without all the nitty gritty 'getting connection' sort of lines along with it

Reply:
if we dont want to see those lines then why log them?
we would want the `tx_id` on the message if it didnt manage to get the connection?